ktor-client/ktor-client-core/common/test/io/ktor/client/features/cache/CacheExpiresTest.kt

```diff
@@ -1,7 +1,8 @@
 /*
  * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
-package io.ktor.client.features.cache.tests
```

Comment:
Why change package name here? Why does it make difference?

Reply:
It does not make any differences. In some tests, you use the normal package name, which allows IDEA to switch from the class right to the test, and sometimes you don't use a package at all, or you use a `tests` package name. In the end, there should be ONE convention in ktor...